modules/support-generating-a-sosreport-archive.adoc

```diff
@@ -77,14 +77,21 @@ If an existing `toolbox` pod is already running, the `toolbox` command outputs `
 ====
 +
 . Collect a `sosreport` archive.
-.. Run the `sosreport` command and enable the `crio.all` and `crio.logs` CRI-O container engine `sosreport` plugins:
+.. Run the `sos report` command and enable the `crio.all` and `crio.logs` CRI-O container engine `sosreport` plugins:
 +
 [source,terminal]
 ----
-# sosreport -k crio.all=on -k crio.logs=on <1>
+# sos report -k crio.all=on -k crio.logs=on <1>
 ----
 <1> `-k` enables you to define `sosreport` plugin parameters outside of the defaults.
 +
+.. Optional: run the following command to include information on OVN-Kubernetes networking configurations from a node in your report:
```

Comment:
```suggestion
.. Optional: To include information on OVN-Kubernetes networking configurations from a node in your report, run the following command:
```

You need the capitalization switch, and I think it's better to put the result of the step first.